test/widgets/profile/edit_gender_page_test.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:motis_mitfahr_app/account/models/profile.dart';
+import 'package:motis_mitfahr_app/account/pages/edit_account/edit_gender_page.dart';
+import 'package:motis_mitfahr_app/account/pages/profile_page.dart';
+import 'package:motis_mitfahr_app/util/supabase.dart';
+import 'package:supabase_flutter/supabase_flutter.dart';
+
+import '../../util/factories/profile_factory.dart';
+import '../../util/mocks/mock_server.dart';
+import '../../util/mocks/request_processor.dart';
+import '../../util/mocks/request_processor.mocks.dart';
+import '../../util/pump_material.dart';
+
+void main() {
+  late Profile profile;
+  final MockRequestProcessor processor = MockRequestProcessor();
+  const String email = 'motismitfahrapp@gmail.com';
+  const String authId = '123';
+  setUpAll(() async {
+    MockServer.setProcessor(processor);
+  });
+
+  setUp(() {
+    profile = ProfileFactory().generateFake(id: 1);
+    SupabaseManager.setCurrentProfile(profile);
+    whenRequest(
+      processor,
+      urlMatcher: startsWith('/auth/v1/token'),
+      bodyMatcher: containsPair('email', 'motismitfahrapp@gmail.com'),
+      methodMatcher: equals('POST'),
+    ).thenReturnJson({
+      'access_token': authId,
+      'token_type': 'bearer',
+      'user': User(
+          id: authId,
+          appMetadata: {},
+          userMetadata: {},
+          aud: 'public',
+          createdAt: DateTime.now().toIso8601String(),
+          email: email),
+      'email': email,
+    });
+    whenRequest(processor, urlMatcher: contains('/rest/v1/profiles')).thenReturnJson(profile.toJsonForApi());
+  });
+  group('edit_gender_page', () {
+    testWidgets('display genderRadioListTile', (WidgetTester tester) async {
+      //load page
+      await pumpMaterial(tester, EditGenderPage(profile));
+      await tester.pump();
+      //check if male is displayed
+      final maleGenderFinder = find.byKey(const Key('0genderRadioListTile'));
+      expect(maleGenderFinder, findsOneWidget);
+
+      //check if female is displayed
+      final femaleGenderFinder = find.byKey(const Key('1genderRadioListTile'));
+      expect(femaleGenderFinder, findsOneWidget);
+
+      //check if diverse is displayed
+      final diverseGenderFinder = find.byKey(const Key('2genderRadioListTile'));
+      expect(diverseGenderFinder, findsOneWidget);
+
+      //check if preferNotToSay is displayed
+      final preferNotToSayGenderFinder = find.byKey(const Key('preferNotToSayGenderRadioListTile'));
+      expect(preferNotToSayGenderFinder, findsOneWidget);
+    });
+    // don't know to test what I have selected
```

Comment:
?

Reply:
I don't think this test is useful anyways? It just tests that the RadioListTile works, which should be guaranteed by Flutter.

Reply:
ok, yea gonna fix that